data/categories.txt

```diff
@@ -5669,7 +5673,7 @@ nl:^Airconditioning|luchtbehandeling
 fi:^Ilmastointilaite
 fr:^Climatiseur
 de:^Heizung|Lüftung|Klima|Installateur
-hu:^Hűtés-fűtés szerelő|légkondicionáló
+hu:^Fűtés-, szellőztetés- és légkondicionálás-szerelő|hűtés- és fűtésszerelő|fűtésszerelő|légkondicionálás-szerelő
```

Comment:
Can we shorten it somehow? Break the first synonym into multiple ones maybe?
It may be too long even for the UI, and I'm sure noone really types exactly such strings when using search.

Reply:
​Yes, the Hungarian translation is very long, even if it is exact. Unfortunately, we don't such a nice abbreviation like that in English. Then let's option for the German solution:
"^Fűtésszerelő|hűtésszerelő|szellőztetésszerelő|​légkondicionálás-szerelő"